package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "cp-local-development",
+  "version": "1.0.0",
+  "repository": "https://github.com/CoderDojo/cp-local-development.git",
+  "author": "CoderDojo Foundation <webteam@coderdojo.com>",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+  	"postinstall": "docker-compose -f docker-compose.yarn.yml up && docker-compose -f docker-compose.yarn.yml down"

```

Comment:
could use to `-f` so `docker-compose.yarn.yml` just overrides parts in `docker-compose.yml` would make the config far smaller and reduce the repetition

Reply:
I tried doing that, but there were some issues IIRC. Plus you get into an issue of overwriting existing created containers. Not a massive issue given you won't be doing this often, but yeah.

Given the current setup just isn't working, and this should, I'd be inclined to fire ahead now and optimise later 😅

Reply:
alright as long as you tried